### DISCUSSION TITLE
Wallets can not transfer out XCH coins

### DISCUSSION THREAD

**Discussion by GStones** [Chia-Network/chia-blockchain#6107] [Support]
![image](https://user-images.githubusercontent.com/6346542/119821482-32afa880-bf25-11eb-8705-d2ce27e3b234.png)

**Comment by GStones** [comment]
ok! it works now! 
thanks

**Comment by BrandtH22** [comment]
@GStones , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
